mlflow/models/evaluation/base.py

```diff
@@ -1435,11 +1435,11 @@ def model(inputs):
                     return pd.DataFrame({"answer": ["bar"], "source": ["baz"]})
 
 
-                results = evaluate(model=model, data=data, predictions="answer", ...)
+                results = evaluate(model=model, data=data, predictions="answer")
```

Comment:
does this work, or does it require extra params that the user has to input? if it requires extra stuff then maybe we can add that in a comment

Reply:
Great suggestion, I'll add a comment for safety.